digital_comms/fixed_network/interventions.py

```diff
@@ -113,20 +98,9 @@ def decide_interventions(system, year, technology, policy, annual_budget,
 
     return built_interventions
 
-def meet_most_beneficial_demand(system, year, technology, policy, annual_budget,
-                                adoption_cap, subsidy, telco_match_funding,
-                                service_obligation_capacity, asset_variable):
-    """Given strategy parameters and a system, meet the most beneficial demand.
-    """
-
-    built_interventions = meet_most_beneficial_demand(distributions, year, technology, policy, annual_budget,
-                                                      adoption_cap, subsidy, telco_match_funding,
-                                                      service_obligation_capacity)
-
-    return built_interventions
 
-def meet_most_beneficial_demand(distributions, year, technology, policy, annual_budget, adoption_cap,
-                                subsidy, telco_match_funding, service_obligation_capacity):
+def meet_most_beneficial_demand(system_assets, year, technology, policy, annual_budget, adoption_cap,
```

Comment:
The docstring needs updating to match the new arguments (need to add `asset_variable` and change `distributions` to `system_assets`)